web_responsive/static/src/css/web_responsive.scss

```
@@ -171,16 +206,27 @@ $chatter_zone_width: 35%;
             }
         }
 
+        .o_app:hover img {
+            transform: translateY(-3px);
+            box-shadow: 0 9px 12px -4px rgba(0, 0, 0, 0.3);
+        }
+
         // Hide app icons when searching
         .has-results ~ .o_app {
             display: none;
         }
 
         .o-app-icon {
             height: auto;
-            max-width: 7rem;
+            max-width: 70px;
```

It's better to stick with `rem` units, which provide a better sense of responsiveness and accessibility. Icons should be proportional to the font size.

However, I agree these could be a little bit smaller:

```suggestion
            max-width: 6rem;
```

---

> Wait until tomorrow to see @Yajo's answer. Right now, I have made a quick test and I see a problem: when you click on the AppSwitcher application, the focus is not put on the search bar, so if you type, you don't get the search.

@pedrobaeza, the problem was occurring due to the transition to display the menu, browsers prevent focus from an element that is not yet visible. I removed the transition to avoid sacrificing functionality.